WordPress/openverse
# Update the box layout of the audio track component

## Fixes



Related to #2910 by @fcoveram

## Description


This PR uses the new `VAudioControl` component (replacing the old `VPlayPause` component) in the box layout of the audio track. This layout is used on the All content search page.
This component is updated to match the requirements in #2910 and the [mockups](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=5622-28655&mode=design&t=KrrG1cFQ6Ur9mrv9-4)

## Testing Instructions

Search for something in "All content" view and check that the audio boxes work as expected: you can play the audio by clicking on the play button or pressing spacebar. If you click on the box itself, you should open the single result page. When you hover on the box, you should see the license.

There are two variants of this layout: one is used on the small screens. It doesn't show the waveforms. If you click on the play button on a small box, you should see the global audio player at the bottom of the screen.

The Audio box layout story should also work and match the Figma mockups.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
